### DISCUSSION TITLE
About reading local files

### DISCUSSION THREAD

**Discussion by summer-shit** [iTwin/itwinjs-core#2133] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/89509530/130743360-0b62cd7d-3812-49e5-9bd2-bcf411affe88.png)
![error](https://user-images.githubusercontent.com/89509530/130743362-f06f1d55-7f1d-47b3-8e59-0a2681853f91.png)
When I run the test case, I want to parse the local file and read the model information. However, it is not clear why this exception is thrown when the file is opened.

**Comment by kabentley** [comment, marked as answer]
It looks like you [have not called `IModelHost.startup`](https://www.itwinjs.org/learning/backend/imodelhost/) in your initialization.

**Comment by summer-shit** [comment]
![error](https://user-images.githubusercontent.com/89509530/130888113-4ece45f5-c66f-41d2-947b-1cc5860e539a.png)

Thank you for your reply. The initialization was added as you said, but this problem has not been solved

**Reply by calebmshafer** [reply]
@summer-shit, `IModelHost.startup` is an async function so you'll want to await it so it can complete before you attempt to open the iModel.

**Reply by summer-shit** [reply]
Thank you very much. The problem has been solved
